llpc/test/shaderdb/general/OpPtrEqualTest.spvasm

```diff
@@ -0,0 +1,86 @@
+; Test OpPtrEqual
```

Comment:
nit: SPIR-V instruction tests are typically in the other test directory: https://github.com/GPUOpen-Drivers/llpc/tree/dev/llpc/test/shaderdb/object

Reply:
Fixed, thanks.